support-workers/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -150,7 +150,7 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
       BankTransferAccountName = sepa.accountHolderName,
       BankTransferAccountNumber = sepa.iban,
       Email = user.primaryEmailAddress,
-      IPAddress = ipAddress,
+      IPAddress = ipAddress.take(15), // zuora doesn't support ipv6
```

Comment:
It feels a little weird to give them a truncated ipv6 string but I guess there's nothing else we really can do

Reply:
Indeed. I'll raise a ticket with zuora later